utilities_common/netstat.py

```diff
@@ -56,6 +56,42 @@ def ns_util(newstr, oldstr, delta, port_rate=PORT_RATE):
         util = rate/(port_rate*1000*1000*1000/8.0)*100
         return "{:.2f}%".format(util)
 
+def format_brate(rate):
+    """
+    Show the byte rate.
+    """
+    if rate == STATUS_NA:
+        return STATUS_NA
+    else:
+        rate = float(rate)
+        if rate > 1024*1024*10:
```

Comment:
Please refer to https://en.wikipedia.org/wiki/Bit_rate
1K should be 1000.

Reply:
fixed